### PR TITLE
feat(ENG-1637): start new sessions from a session

### PR DESCRIPTION
**Pull request by itayd**
this adds the `start` syscall to python

### CODE REVIEW

**Comment by efiShtain** [comment]
do we really need this capability? 
does the UI knows how to handle multi session stuff ?
can't we just trigger a different webhook and let it run as any other workflow ?
do we have any relation between this sessions ? how do we handle replay ? how do we handle error in the child sessions ?

**Review by efiShtain — APPROVED** [review]
fix formatting before merge

File: runtimes/pythonrt/runner/syscalls.py [hunk 1]
```diff
@@ -36,6 +37,33 @@ def call(self, fn, args, kw):
...
+            json_data = json.dumps(data)
+        except (ValueError, TypeError) as err:
+            raise ValueError(f"start: invalid data: {err}")
+
+        try:
+            json_memo = json.dumps(memo)
+        except (ValueError, TypeError) as err:
+            raise ValueError(f"start: invalid memo: {err}")
+
+        req = pb.StartSessionRequest(runner_id=self.runner_id, location=loc, data=bytearray(json_data), memo=bytearray(json_memo))
```
Comment (tebeka) [inline runtimes/pythonrt/runner/syscalls.py]:
```suggestion
        req = pb.StartSessionRequest(runner_id=self.runner_id, location=loc, data=json_data.encode(), memo=json_memo.encode())
```

**Review by tebeka — APPROVED** [review]

File: runtimes/pythonrt/runner/syscalls.py (hunk 1 above)
Comment (itayd) [reply runtimes/pythonrt/runner/syscalls.py]: done

**Pull request merged**